README.md

```diff
@@ -82,6 +82,17 @@ should be called to update the UI.
 This in turn will invoke the `build` method 
 which re-renders the widget.
 
+## Flutter application test
+
+As in all languages or platforms the secret to a successful application is to test it.
+Several tests should be done like the functionality test, the user experience test, which can help us correct any bug.</br>
+
+An application must have tested any function, any class or task needed to run correctly without errors.</br>
```

Comment:
```suggestion
An application must have tested all functions, classes or tasks needed to run correctly without errors.</br>
```
"any" is too open to interpretation. "all" is specific and encompasses _all_ the code.
the difference may seem subtle but it's important to chose your words carefully. 
see: https://www.ietf.org/rfc/rfc2119.txt